my-project/src/components/Field/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './style.scss';
+
+const myInput = (props) => {
+    const { input, type, placeholder, meta } = props;
+    return (
+        <>
+            <input {...props.input} type={props.type} placeholder={props.placeholder} />
+            {meta.error &&
```

Comment:
It's better to use descructurization for avoid duplicate `meta.`:
`const myInput = (input, type, placeholder, meta: { error, touched })`

Reply:
done